requirements.txt

```diff
@@ -8,6 +8,6 @@
 #
 # PBR should always appear first
 pbr!=2.1.0,>=2.0.0 # Apache-2.0
--e git+https://github.com/ChameleonCloud/python-blazarclient.git@chameleoncloud/stable/train#egg=python_blazarclient-2.1.0
+-e git+https://github.com/ChameleonCloud/python-blazarclient.git@resource-properties-api#egg=python_blazarclient-2.1.0
```

Comment:
Is this already merged to the Train branch? If so, let's avoid changing requirements here.